src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import original from 'react95/dist/themes/original';
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import {
```

Comment:
Consider using styled-components for styling instead of SCSS. This will make it easier to manage styles and improve performance. 💅

Reply:
> improve performance

Lol.. I am disappointed in ChatGPT🥲